SprintDataset.py

```diff
@@ -79,6 +80,10 @@ def __init__(self, target_maps=None, str_add_final_zero=False, input_stddev=1.,
     if bpe:
       from GeneratingDataset import BytePairEncoding
       self.bpe = BytePairEncoding(**bpe)
+    if orth_vocab:
```

Comment:
Something like `self.orth_vocab = None` is missing before the `if`. Also see the failed test case.